gtk/MessageLogWindow.cc

```diff
@@ -324,7 +326,7 @@ void renderText(
 void renderTime(Gtk::CellRendererText* renderer, Gtk::TreeModel::const_iterator const& iter)
 {
     auto const* const node = iter->get_value(message_log_cols.tr_msg);
-    renderer->property_text() = Glib::DateTime::create_now_local(node->when).format("%T");
+    renderer->property_text() = Glib::DateTime::create_now_local(std::chrono::system_clock::to_time_t(node->when)).format("%T");
```

Comment:
```suggestion
    renderer->property_text() = Glib::DateTime::create_now_local(system_clock::to_time_t(node->when)).format("%T");
```

Reply:
This is in another namespace that does not have `using std::chrono::xxx`.